src/rust/sdk/src/tx_builder.rs

```diff
@@ -490,6 +622,12 @@ impl<'a> EconiaTransactionBuilder<'a> {
         self
     }
 
+    /// Set the internal [EntryFunction] as [register_market_account_generic_base](https://github.com/econia-labs/econia/blob/main/src/move/econia/doc/user.md#0xc0deb00c_user_register_market_account_generic_base)
+    ///
+    /// Arguments:
+    /// * `quote`: Aptos [TypeTag] for quote coin.
+    /// * `market_id`: Market ID for corresponding market.
+    /// * `custodian_id`: Id of market custodian.
```

Comment:
This is inconsistent: `ID` in the first line then `Id` in the second. I suspect there are other inconsistencies in the file.